### PR TITLE
[DSD-1009] added required contents to resident-ui deployment

### PR DESCRIPTION
**Pull request by Rakshitha650**

### CODE REVIEW

File: charts/resident-ui/values.yaml [hunk 1]
```diff
@@ -410,8 +410,13 @@ metrics:
...
     #       severity: error
     rules: []
+ingress:
+  enabled: false
+
+autoscaling:
+  enabled: false
 
-resident-ui:
+residentUi:
```
Comment (syed-salman-technoforte) [inline charts/resident-ui/values.yaml:419]: change this to residentUi to resident

File: charts/resident/templates/virtualservice.yaml [hunk 2]
```diff
@@ -25,6 +25,8 @@ spec:
         host: {{ template "common.names.fullname" . }}
         port:
           number: {{ .Values.service.port }}
+    corsPolicy:
+    { { - include "common.tplvalues.render" ( dict "value" .Values.istio.corsPolicy "context" $ ) | nindent 6 } }
```
Comment (syed-salman-technoforte) [inline charts/resident/templates/virtualservice.yaml:29]: change {  {    to  {{ 
change }  }    to  }}

File: charts/resident/values.yaml [hunk 3]
```diff
@@ -424,3 +424,34 @@ istio:
     - istio-system/public
     - istio-system/internal
   prefix: /resident/v1
+  corsPolicy:
+    allowOrigins:
+      - prefix: https://resident-ui.sandbox.xyz.net
```
Comment (syed-salman-technoforte) [inline charts/resident/values.yaml:429]: change https://resident-ui.sandbox.xyz.net 
to https://resident.sandbox.xyz.net

File: charts/resident-ui/templates/NOTES.txt [hunk 4]
```diff
@@ -1,22 +0,0 @@
-1. Get the application URL by running these commands:
```
Comment (syed-salman-technoforte) [inline charts/resident-ui/templates/NOTES.txt:1]: why this was removed

**Review by syed-salman-technoforte — CHANGES_REQUESTED** [review]

File: charts/resident-ui/templates/NOTES.txt (hunk 4 above)
Comment (Rakshitha650) [reply charts/resident-ui/templates/NOTES.txt:1]: Lalith told its not much required we don't have this file in some modules

File: charts/resident-ui/templates/configmap.yaml [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+    {{- if .Values.commonLabels }}
+    {{- include "common.tplvalues.render" ( dict "value" .Values.commonLabels "context" $ ) | nindent 4 }}
+    {{- end }}
+  {{- if .Values.commonAnnotations }}
+  annotations: {{- include "common.tplvalues.render" ( dict "value" .Values.commonAnnotations "context" $ ) | nindent 4 }}
+  {{- end }}
+
+data:
+  config.json: |-
+   {"baseUrl":"https://{{ .Values.residentUi.apiHost }}/resident/v1/"}
```
Comment (syed-salman-technoforte) [inline charts/resident-ui/templates/configmap.yaml:16]: change .Values.residentUi to .Values.resident

File: charts/resident-ui/values.yaml [hunk 6]
```diff
@@ -410,8 +410,13 @@ metrics:
...
     rules: []
+ingress:
+  enabled: false
+
+autoscaling:
+  enabled: false
 
-resident-ui:
+resident:
   apiHost: resident-ui.sandbox.xyz.net
```
Comment (syed-salman-technoforte) [inline charts/resident-ui/values.yaml:420]: this is api Host so it should be api-internal.sandbox.xyz.net

**Review by syed-salman-technoforte — CHANGES_REQUESTED** [review]

**Review by syed-salman-technoforte — APPROVED** [review]

**Pull request merged**